### ISSUE TITLE
TypeError: unhashable type: 'list'

### ISSUE THREAD

**Issue by ArtiVlasov** [uparma/uparma-py#10]
get_default_params() returns the error if the translated key is a list. Realised while working on the omssa wrapper, that e.g. parameter 135 returns a list for omssa_style_1, which cannot be handled.

Might not be the only case.

The error is raised in the following line:
https://github.com/uparma/uparma-py/blob/ca40db2f31248badc8f7a9fe24d3bfe2ad1df6fd/uparma/uparma.py#L439

Update
======

`'-tem'` and `'-tom'` are only used when `label` is set to 15N:

```
if self.params['translations']['label'] == '15N':
            translations['-tem']['precursor_mass_type'] = '2'
            translations['-tom']['frag_mass_type'] = '2'
```

This is the parameter description from the [omssa page](https://proteomicsresource.washington.edu/protocols06/omssa.php)

```
 -tom <Integer>
   product ion search type (0 = mono, 1 = avg, 2 = N15, 3 = exact)
   Default = `0'
 -tem <Integer>
   precursor ion search type (0 = mono, 1 = avg, 2 = N15, 3 = exact, 4 =
   multiisotope)
   Default = `0'
```

so basically, the value translation should be 0 for 14N and 2 for 15N, right?

**Comment by MKoesters** [comment]
I added a test to reproduce this issue https://github.com/uparma/uparma-py/tree/fix/%2310
